include/pistache/date.h

```diff
@@ -4576,10 +4577,10 @@ to_stream(std::basic_ostream<CharT, Traits>& os, const CharT* fmt,
                         auto d = static_cast<int>(static_cast<unsigned>(fds.ymd.day()));
                         if (d < 10)
                             os << ' ';
-                        os << d << ' ' 
+                        os << d << ' '
```

Comment:
Please do not change whitespace/formatting in the same PR as a logic change.

Reply:
Yes that happened unintentionally, my editor removes trailing whitespaces automatically. I thought it would be OK to leave, but will remove it.
Thanks for the review!